### PR TITLE
[1119] Fix cookie overflow

### PR DESCRIPTION
**Pull request by mooktakim**
### Context

https://trello.com/c/jjZUDyqQ/1119-m-fix-cookie-overflow

### Changes proposed in this pull request

* Replace session store cookie_store with active_record_store, gem `activerecord-session_store`
* New ActiveJob that truncates the table of rows older than 30 days (by default, can be changed with ENV variable `SESSION_DAYS_TRIM_THRESHOLD`)
* ActiveJob runs nightly (midnight), added to sidekiq-cron.
* New migration to create "sessions" table.

### Guidance to review
* Gem URL: https://github.com/rails/activerecord-session_store

### CODE REVIEW

**Comment by mooktakim** [comment]
[[M] Fix cookie overflow](https://trello.com/c/jjZUDyqQ/1119-m-fix-cookie-overflow)

File: app/jobs/session_store_truncate_job.rb [hunk 1]
```diff
@@ -0,0 +1,11 @@
+# frozen_string_literal: true
+
+class SessionStoreTruncateJob < ApplicationJob
+  queue_as :default
+
+  def perform
+    # Copying from: rake db:sessions:trim
+    cutoff_period = (ENV["SESSION_DAYS_TRIM_THRESHOLD"] || 30).to_i.days.ago
```
Comment (gpeng) [inline app/jobs/session_store_truncate_job.rb:8]: Can we put this in settings.yml and use `Settings.session_expiry_period` or something?

**Review by gpeng — CHANGES_REQUESTED** [review]

File: app/jobs/session_store_truncate_job.rb (hunk 1 above)
Comment (mooktakim) [reply app/jobs/session_store_truncate_job.rb:8]: Will add that now

**Review by gpeng — CHANGES_REQUESTED** [review]

**Review by gpeng — APPROVED** [review]

**Pull request merged**